packages/@functionless/test/src/runtime.ts

```diff
@@ -1,3 +1,5 @@
+import "jest";
+
```

Comment:
you should not need this.

Reply:
I was getting errors without it. Let me try agian.